examples/HSE/working-at-height.xml

```diff
@@ -0,0 +1,249 @@
+      <!-- GDS taxonomy topics -->
+      <regulatoryTopic>Work</regulatoryTopic>
+      <regulatoryTopic>Health and safety at work</regulatoryTopic>
+      <status>published</status>
+      <!-- mandatory metadata - end -->
+    </orp>
+  </metadata>
+  <documentContent>
+    <html>
+      <body>
+        <article aria-label="Working at Height"> 
+            <h1>Working at height</h1> 
+          <h2>A brief guide</h2>
+          <section aria-label="introduction"> 
+            <h3>Introduction</h3>
+            <p>This brief guide describes what you, as an employer, need to do to protect your employees from falls from height. It will also be useful to employees and their representatives. Following this guidance is normally enough to comply with the Work at Height Regulations 2005 (WAHR). You are free to take other action, except where the guidance says you must do something specific. Falls from height are one of the biggest causes of workplace fatalities and major injuries. Common causes are falls from ladders and through fragile roofs. The purpose of WAHR is to prevent death and injury from a fall from height. Work at height means work in any place where, if there were no precautions in place, a person could fall a distance liable to cause personal injury. For example you are working at height if you:</p> 
+            <ul>
+              <li>are working on a ladder or a flat roof&#59;</li>
+              <li>could fall through a fragile surface&#59;</li>
+              <li>could fall into an opening in a floor or a hole in the ground.</li>
+            </ul>
+            <p>Take a sensible approach when considering precautions for work at height. There may be some low-risk situations where common sense tells you no particular precautions are necessary and the law recognises this. There is a common misconception that ladders and stepladders are banned, but this is not the case. There are many situations where a ladder is the most suitable equipment for working at height. Before working at height you must work through these simple steps:</p>
+            <ul>
+              <li>avoid work at height where it is reasonably practicable to do so&#59;</li>
+              <li>where work at height cannot be avoided, prevent falls using either an existing place of work that is already safe or the right type of equipment&#59;</li>
+              <li>minimise the distance and consequences of a fall, by using the right type of equipment where the risk cannot be eliminated.</li>
+            </ul>
+            <p>Figure 1 gives further guidance and examples for each of the above steps to help you comply with the law. You should:</p>
+             <ul>
+              <li>do as much work as possible from the ground&#59;</li>
+              <li> ensure workers can get safely to and from where they work at height&#59;</li>
+              <li>■ ensure equipment is suitable, stable and strong enough for the job, maintained and checked regularly&#59;</li>
```

Comment:
Is this black box a typo? - same for line 94

Reply:
@kevinxufs yes just a formatting issue, I've pushed up a fix.